CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+##[3.0.0] 2024-05-02
+
+### Changed
+- React Native version updated to 0.74.0
+- `init` deprecated method - only method declaration lefted;
```

Comment:
Let's update the description a bit (for all the things bellow):
`init` - no longer supported;

Reply:
done